src/qcodes_contrib_drivers/drivers/Kiutra/L_Type_Rapid.py

```
@@ -0,0 +1,423 @@
+        Args:
+            value: The ramp rate value to validate.
+            context: A string providing context for the validation.
+
+        Raises:
+            TypeError: If the value is not a valid number.
+            ValueError: If the ramp rate is outside the allowed range for the
+                current setpoint.
+        """
+
+        if not isinstance(value, (int, float, np.integer, np.floating)):
+            raise TypeError(f"{value!r} is not an int or float; {context}")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)